.gitlab-ci.yml

```diff
@@ -44,7 +44,15 @@ analysis:licenses:
   stage: analysis
   script:
     - git fetch --depth=1 origin master
-    - ./gradlew :dd-sdk-android:checkThirdPartyLicences
+    - ./gradlew :dd-sdk-android:checkThirdPartyLicences :dd-sdk-android-timber:checkThirdPartyLicences
+
+analysis:api-surface:
+  tags: [ "runner:docker", "size:large" ]
+  image: $CI_IMAGE_DOCKER
+  stage: analysis
+  script:
+    - git fetch --depth=1 origin master
```

Comment:
This might keep giving false-positives on long-living feature stabilisation branches (like tracing). How about using `CI_MERGE_REQUEST_TARGET_BRANCH_NAME` GitLab variable?

Reply:
Because we don't have this value available. Git lab runs on commits and tags only, it's never triggered on pull requests.